stack-8.8.4.yaml

```
@@ -0,0 +1,7 @@
+resolver: lts-16.2
```

this resolver is based on ghc 8.8.3 (as you can see on its page: https://www.stackage.org/lts-16.2)
so the 8.8.4 in the file name is a bit misleading.
I'd just change this line to 
```suggestion
resolver: lts-16.31
```
which is the latest 8.8.4 based resolver